omnigibson/scenes/scene_base.py

```diff
@@ -59,20 +59,35 @@ def __init__(
         self,
         scene_file=None,
         use_floor_plane=True,
+        floor_plane_size=1000.0,
         floor_plane_visible=True,
         floor_plane_color=(1.0, 1.0, 1.0),
+        floor_plane_grid=False,
+        floor_plane_grid_resolutions=(0.001, 0.01, 0.1, 1.0),
         use_skybox=True,
+        use_skybox_texture=True,
+        skybox_color_temperature=5000,
     ):
         """
         Args:
             scene_file (None or str or dict): If specified, full path of JSON file to load (with .json) or the
                 pre-loaded scene state from that json.
                 None results in no additional objects being loaded into the scene
             use_floor_plane (bool): whether to load a flat floor plane into the simulator
+            floor_plane_size (float): size of the visible floor plane to load into the simulator
             floor_plane_visible (bool): whether to render the additionally added floor plane
             floor_plane_color (3-array): if @floor_plane_visible is True, this determines the (R,G,B) color assigned
                 to the generated floor plane
+            floor_plane_grid (bool): whether to render a checkerboard floor plane with grid squares
+            floor_plane_grid_resolutions (Tuple[float]): A tuple of integers where each integer represents a
+                grid cell size, e.g. (0.001, 0.01, 1.) will render a floor plane with 3 different grid cell sizes at 1mm,
+                1cm, and 1m respectively. Will be ignored if @floor_plane_grid is False.
             use_skybox (bool): whether to load a skybox into the simulator
+            use_skybox_texture (Union[str, bool]): whether to use a texture for the skybox or not. False will disable the
+                texture and expose the skybox color, True will use the default texture, and a string will use the
+                specified texture.
+            skybox_color_temperature (float): Color temperature of the skybox if @use_skybox is True. This is the color of the
+                light emitted by the skybox in Kelvin, and the color of the skybox itself if no texture is used.
```

Comment:
This is also a bit confusing. It's the color of the skybox AND it's the color of the light emitted by the skybox? Shouldn't we then split these into two variables in case they might be different in the future?

Reply:
There isn't a way to separate these two things currently. The skybox emits light in its color.